package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "etherspot",
-  "version": "1.3.10",
+  "version": "1.3.11",
```

Comment:
I think it should be 1.4.1

Reply:
Thanks @jsidorenko, I've changed the version to 1.4.1 !